src/components/core/geoChart/geoChartDataLayers.ts

```diff
@@ -17,54 +17,56 @@ import {
     PUSHPIN_STYLE_CIRCLE_SIZE,
     PUSHPIN_STYLE_CIRCLE_STROKE_COLOR,
     EMPTY_SEGMENT_VALUE,
+    PUSHPIN_SIZE_OPTIONS_MAP,
 } from "../../../constants/geoChart";
-import { IGeoData, IGeoPushPinConfig } from "../../../interfaces/GeoChart";
+import { IGeoData, IGeoPointsConfig, IGeoConfig } from "../../../interfaces/GeoChart";
 import { SEGMENT } from "../../../constants/bucketNames";
 import { getMinMax } from "../../../helpers/utils";
-import { pushpinSizeOptionsMap } from "./geoChartDataSource";
+import isEmpty = require("lodash/isEmpty");
 
 function getExpressionByBucketName(name: string): mapboxgl.Expression {
     return ["get", "value", ["object", ["get", name]]];
 }
 
 function createPushpinSizeOptions(
     geoData: IGeoData,
-    pushpinSizeOptions: IGeoPushPinConfig,
+    pushpinSizeOptions: IGeoPointsConfig,
```

Comment:
This `pushpinSizeOptions` should be renamed to `geoPointsConfig` to match its interface name, shouldn't it?